app.js

```diff
@@ -15,6 +18,8 @@ app.get("/api/reviews", getReviews);
 
 app.get("/api/reviews/:review_id/comments", getCommentsByReviewId);
 
+app.patch("/api/reviews/:review_id", patchReviewById);
+
 app.use((err, req, res, next) => {
```

Comment:
you have not yet added a 500 error handler, I think its almost certainly making debugging your code harder than it needs to be, I would advise you to add one :)